jetty-ee10/jetty-ee10-annotations/src/main/java/org/eclipse/jetty/ee10/annotations/AnnotationParser.java

```diff
@@ -98,7 +90,7 @@ private static int asmVersion()
             .map(Integer::parseInt)
             .max(Integer::compareTo);
 
-        if (!asmVersion.isPresent())
+        if (asmVersion.isEmpty())
```

Comment:
I say potayto you say potahto.

Reply:
It's the same end result, but easier to read.